documentation/book/con-customizing-pod-disruption-budgets.adoc

```diff
@@ -0,0 +1,33 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-customizing-deployments.adoc
+
+[id='con-customizing-pod-disruption-budgets-{context}']
+= Customizing Pod Disruption Budgets
+
```

Comment:
_Please reword the first paragraph as follows:_

AMQ Streams creates a pod disruption budget for every new `StatefulSet` or `Deployment`. By default, these pod disruption budgets only allow a single pod to be unavailable at a given time by setting the `maxUnavailable` value in the`PodDisruptionBudget.spec` resource to 1. You can increase the amount of unavailable pods allowed by using the pod disruption budget template, which applies to each type of cluster (Kafka and Zookeeper; Kafka Connect and Kafka Connect with S2I support; and Kafka Mirror Maker).

Reply:
> You can increase the amount of unavailable pods allowed by changing the default value of maxUnavailable in the pod disruption budget template.

The budget can be increased as well as decreased. So I changed _increased_ to _change_. I hope this is OK for you.